src/containers/LandingPage.jsx

```diff
@@ -1,6 +1,14 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { openModalAction } from '../actions';
 
 export class LandingPage extends Component {
+  componentDidMount() {
+    const { props: { history: { action } } } = this;
+    const { props: { openModal } } = this;
+    if (action === 'REPLACE') openModal();
```

Comment:
Hi @TheDrizzyWay Please can you explain what's going on here?

Reply:
This ensures that the login modal automatically opens up on redirect. The history object has a key called 'action' which gets a value of 'REPLACE' whenever a redirect is performed.